app/src/sharedTest/java/org/oppia/app/parser/FractionInteractionViewModelTest.kt

```diff
@@ -0,0 +1,155 @@
+package org.oppia.app.parser
+
+import android.content.Context
+import android.content.res.Configuration
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.google.common.truth.Truth.assertThat
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.oppia.app.model.InteractionObject
+import org.oppia.app.model.UserAnswer
+
+@RunWith(AndroidJUnit4::class)
+class StringToNumberWithUnitsParserTest {
+
+  private var context: Context? = null
+  @Before
```

Comment:
Nit: there should be vertical whitespace between a property list and methods per https://developer.android.com/kotlin/style-guide#vertical.